src/Microsoft.Health.Dicom.Functions/Microsoft.Health.Dicom.Functions.csproj

```diff
@@ -33,6 +33,12 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Remove="Registration\**" />
+    <EmbeddedResource Remove="Registration\**" />
```

Comment:
guess these are unintended lines?

Reply:
Yes, I've removed these! They somehow snuck in there